teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(_id, age, specialization, parent_permission: true, name: 'Unknown')
```

Comment:
- [ ] Also remove `attr_reader :specialization` and `id` from initialize 👍 as it;ll be generated automatically, It'll be something like this:

```ruby
class Teacher < Person
 
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
  
  def can_use_services?
    true
  end
end
```

Kindly fix it 👍